test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -176,3 +176,46 @@
                 "message" (str "No status function with version 3 "
                             "found for service foo")}
               (json/parse-string (slurp (:body resp)))))))))
+
+(defn response->status
+  [resp]
+  (:status (json/parse-string (slurp (:body resp)) true)))
+
+(deftest level>=test
+  (testing "use of level>= to implement a status function"
+    (let [my-status (fn [level]
+                      (let [level>= (partial status-core/level>= level)]
+                        {:is-running true
+                         :status (cond-> {:this-is-critical "foo"}
+                                   (level>= :info) (assoc :bar "bar"
+                                                          :baz "baz")
+                                   (level>= :debug) (assoc :x "x"
+                                                           :y "y"
+                                                           :z "y"))}))
```

Comment:
Yeah, don't really love nor hate any of those.  `level->numeric` or `level->int` or `level-as-int`, I'm probably ok with any of those.  I think maybe `level->int` is my favorite.